### ISSUE TITLE
Request for feedback on Treebeard

### ISSUE THREAD

**Issue by alex-remedios** [spatialaudio/digital-signal-processing-lecture#8]
Hey folks,

I noticed you tried out [treebeard](https://github.com/treebeardtech/treebeard) the other day but decided it wasn't for you. Also that you are the developers of nbsphinx!

I'd love to hear any misconceptions/frustrations you had about treebeard, as I am currently prototyping a [lower-level](https://github.com/treebeardtech/nbmake) pytest plugin for parallel testing + building jupyterbooks.

Any feedback is really valuable, thanks.

**Comment by spors** [comment]
Dear Alex,

thanks for reaching out. I really like treebeard and will definitely use it for other projects.

My original idea was to move the already running CI from travis-ci to github workflows. We developed nbsphinx in order to be able to compose documentation from a series of jupyter notebooks. I use it for my Open Educational Resources to generate a single PDF, a book, from the collection of notebooks. Since the notebooks can be executed in this process, this turned out to be also helpful for realizing CI for the collection Jupyter notebooks.

When playing around with juypter workflows these days, I tried out treeboard first and liked it very much. The only issue I ran into was that one notebook cannot be executed on a virtual machine since it requires access to the sound card. In nbsphinx I can exclude single cells or a notebook from execution by editing the notebook/cell metadata. I was planning to open-up an issue for treeboard asking for a possibility to exclude single notebooks from being executed. This could be an interesting feature for other users.

Greetings,
Sascha
--
ScientistsforFuture, https://www.scientists4future.org/

Von: Alex Remedios <notifications@github.com>
Antworten an: spatialaudio/digital-signal-processing-lecture <reply@reply.github.com>
Datum: Mittwoch, 25. November 2020 um 10:20
An: spatialaudio/digital-signal-processing-lecture <digital-signal-processing-lecture@noreply.github.com>
Cc: Subscribed <subscribed@noreply.github.com>
Betreff: [spatialaudio/digital-signal-processing-lecture] Request for feedback on Treebeard (#8)


Hey folks,

I noticed you tried out treebeard<https://github.com/treebeardtech/treebeard> the other day but decided it wasn't for you. Also that you are the developers of nbsphinx!

I'd love to hear any misconceptions/frustrations you had about treebeard, as I am currently prototyping a lower-level<https://github.com/treebeardtech/nbmake> pytest plugin for parallel testing + building jupyterbooks.

Any feedback is really valuable, thanks.
